### PR TITLE
Implement trivial inverses for some more atoms

### PR DESCRIPTION
**Pull request by akshayka**
Edge case in which a user takes a sum/cumsum of a scalar atom.

### CODE REVIEW

**Comment by akshayka** [comment]
Some context: The basic issue is that DQCP allows the root of an expression tree to be transformed by monotone increasing (or decreasing) real functions. The reduction just inverts these functions. Looks like we missed a few inverses in the original implementation.

This part of the grammar is silly, though, because all such a transformation does is yield an equivalent problem. We included it for completeness, but if I were to do it again I'd not allow monotone transformations. Allowing them adds little value, but adds significant maintenance burden ... Perhaps something for CVXPY 2.0 ...

File: cvxpy/reductions/dqcp2dcp/inverse.py [hunk 1]
```diff
@@ -17,11 +17,12 @@
 from cvxpy.atoms.affine.add_expr import AddExpression
 from cvxpy.atoms.affine.binary_operators import DivExpression
 from cvxpy.atoms.affine.unary_operators import NegExpression
+from cvxpy.atoms.affine.sum import Sum
 import numpy as np
 
 INVERTIBLE = set(
     [atoms.ceil, atoms.floor, NegExpression, atoms.exp, atoms.log, atoms.log1p,
-     atoms.logistic, atoms.power, atoms.abs])
+     atoms.logistic, atoms.power, atoms.abs, atoms.cumsum])
```
Comment (SteveDiamond) [inline cvxpy/reductions/dqcp2dcp/inverse.py:25]: Why no sum here?

**Review by SteveDiamond — APPROVED** [review]
LGTM

File: cvxpy/reductions/dqcp2dcp/inverse.py (hunk 1 above)
Comment (akshayka) [reply cvxpy/reductions/dqcp2dcp/inverse.py]: Added some comments.

**Review by SteveDiamond — APPROVED** [review]

**Pull request merged**